File: Compiler/Define/stringToKind.cpp
#include <string>
#include <map>
#include "Kind.h"
using namespace std;

static map<wstring, Kind> stringToKind = {
    {L"#unknown",       Kind::Unknown},
    {L"#EndOfCode",     Kind::EndOfCode},

    {L"참",             Kind::TrueLiteral},
    {L"거짓",           Kind::FalseLiteral},
    {L"#Number",        Kind::NumberLiteral},
    {L"#String",        Kind::StringLiteral},
    {L"#identifier",    Kind::Identifier},

    {L"함수",           Kind::Function},
    {L"반환한다",       Kind::Return},
    {L"만약",           Kind::If},
    {L"아니면",         Kind::Else}
};

File: CS_Compiler/Code/Define/stringToKind.cs
namespace Define {
    class STK {
        public static readonly Dictionary<string, Kind> stringToKind = new()
        {
            {"#unknown",       Kind.Unknown},
            {"#EndOfCode",     Kind.EndOfCode},

            {"참",             Kind.TrueLiteral},
            {"거짓",           Kind.FalseLiteral},
            {"#Number",        Kind.NumberLiteral},
            {"#String",        Kind.StringLiteral},
            {"#identifier",    Kind.Identifier},

            {"함수",           Kind.Function},
            {"반환한다",       Kind.Return},
            {"만약",           Kind.If},
            {"아니면",         Kind.Else}
        };
    }
}